scripts/tempesta.sh

```diff
@@ -258,6 +300,26 @@ start_tempesta_and_check_state()
 	fi
 }
 
+start_tfw_logger()
+{
+	if [ $tfw_logger_should_start -eq 0 ]; then
+		return
+	fi
+
+	if [ -z "$mmap_host" ] || [ -z "$mmap_log" ]; then
+		error "You need to specify 'mmap_host' and 'mmap_log' "`
+		      `"if access_log mmap was specified"
+		return
+	fi
+
+	utils/tfw_logger -H "$mmap_host" -l "$mmap_log" -u "$mmap_user" -p "$mmap_password"
```

Comment:
@krizhanovsky should we check that start of logger was successful? I think that we should at least write error if it fails